tests-functional/pages/about_home.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+
+
+class AboutHome(Base):
```

Comment:
I suspect you don't want a Firefox `about:` page to extend from `Base`. Ultimately you'll hopefully just be able to depend on a package that provides objects for these pages, but until then I would recommend inheriting directly from `pypom.Page`.